### DISCUSSION TITLE
Controller mapping for 'B' button is off

### DISCUSSION THREAD

**Discussion by B1gdaddybr00x** [mupen64plus-ae/mupen64plus-ae#1183] [Q&A]
On Mupen64 with an Xb9x One controller linked via bluetooth, no matter how many times I re-map the 'B' Button it operates as though I presswled the back button of the phone itself with no input on the game. How can I remedy this?

**Comment by ihoid123** [comment]
Same Problem with 8bitdo 64 bt Controller.
